group_project/assignment_6.py

```diff
@@ -7,9 +7,20 @@
 O  |  X  |  O
 """
 
-
+BOARD_TEMPLATE = """
+{}  |  {}  |  {}
+--------------
+{}  |  {}  |  {}
+--------------
+{}  |  {}  |  {}
+"""
+ 
 def format_tic_tac_toe_board(first_row, second_row, third_row):
-    pass
+    return BOARD_TEMPLATE.format(
```

Comment:
This is great! Straightforward replacement of arguments that's easy to visualise and understand.